src/main/java/de/komoot/photon/query/PhotonRequestFactory.java

```diff
@@ -19,7 +19,8 @@
     private final LanguageChecker languageChecker;
     private final static GeometryFactory geometryFactory = new GeometryFactory(new PrecisionModel(), 4326);
 
-    protected static HashSet<String> m_hsRequestQueryParams = new HashSet<>(Arrays.asList("lang", "q", "lon", "lat", "limit", "distance_sort", "osm_tag"));
+    protected static HashSet<String> m_hsRequestQueryParams = new HashSet<>(Arrays.asList("lang", "q", "lon", "lat",
+            "limit", "distance_sort", "osm_tag", "location_bias_scale"));
```

Comment:
Can distance_sort be removed here?

Reply:
Ah, yeah had it removed locally already.